pkg/cmd/codespace/create.go

```diff
@@ -265,6 +271,14 @@ func getMachineName(ctx context.Context, apiClient apiClient, repoID int, machin
 	return selectedMachine.Name, nil
 }
 
+func getRepoSuggestions(ctx context.Context, apiClient apiClient, partialSearch string) []string {
```

Comment:
Nitpick: there doesn't seem to be much value added by this function, so I would suggest that this get inlined at call point

Reply:
The reason I pulled it out into a separate method is for later improvements with some ideas I left in the source issue (#5093) such as including repos that have recently been used by the cli - e.g. repos the user commonly creates codespaces for. This method allows us to build up that more complicated logic in this method rather than inlined in the Survey config. That said, it's easy to pull it back out into a method later also so I'm OK to inline it.

Reply:
Actually I take that back - now that it the `per_page` and `sort` params are refactored out, the method is a little bit big for inlining